src/webserver/WebServer.jl

```diff
@@ -69,7 +69,7 @@ function run(; kwargs...)
 end
 
 function run(options::Configuration.Options)
-    session = ServerSession(;options=options)
```

Comment:
This change makes no difference in Julia >= 1.3, but would break earlier Julia versions. As Pluto only supports Julia >= 1.6, this is however fine.